ui/org.eclipse.pde.ui/src/org/eclipse/pde/internal/ui/editor/build/BuildSourceViewerConfiguration.java

```diff
@@ -241,6 +240,7 @@ public boolean affectsColorPresentation(PropertyChangeEvent event) {
 
 	@Override
 	public IContentAssistant getContentAssistant(ISourceViewer sourceViewer) {
+		BuildPropertiesContentAssistProcessor fContentAssistantProcessor;
```

Comment:
This can be moved down to the only location where the variable is assigned.

Reply:
@HannesWell - thanks, brought it to the lowest possible point, and converted to a `var style declaration` as per the `quick-fix` suggestion that appeared in the context.